components/user_list_row.jsx

```diff
@@ -13,98 +13,101 @@ import * as Utils from 'utils/utils.jsx';
 
```

Comment:
Made this into a component. None of the code changes, it's just been reorganized